### DISCUSSION TITLE
Export CSV

### DISCUSSION THREAD

**Discussion by thuyvo3101** [eclipse-birt/birt#1023] [Q&A]
Hello,

I have 3 tables in my reports. i want to export those 3 tables in one CSV file. Is BIRT able to do that? Currently, BIRT UI just allowed export 1 table to CSV.

Thank you everyone!

Thuy

**Comment by hvbtup** [comment]
One possible workaround is to use Javascript to create a CSV file.
The idea is to use a `java.io.FileWriter( some_filename )`, then use its `write` method to write text out (of course you have to handle escaping, seperation characters etc yourself), and close it.
Call the Javascript code eg eg from afterOpen, onFetch, afterClose events of your data sets.
Note that this will create a CSV file *in addition* to the normal (eg PDF) output file.

Besides, 3 tables in one CSV only make sense if all 3 of the have an identical column layout.
